src/schema/artwork/__tests__/artwork.test.js

```diff
@@ -1286,7 +1286,20 @@ describe("Artwork type", () => {
       }
     `
 
+    it("is null if artwork is not enrolled in in an e-commerce program", () => {
+      artwork.acquireable = false
```

Comment:
I guess for this test to actually test a real case the artwork should have the `domestic_shipping_fee_cents` and `international_shipping_fee_cents` set and still return null in this case. The way it's written right now it will probably return null even without your change.